qualtran/linalg/lcu_util.py

```diff
@@ -147,7 +147,7 @@ def _preprocess_for_efficient_roulette_selection(discretized_probabilities):
 
 
 def preprocess_lcu_coefficients_for_reversible_sampling(
-    lcu_coefficients: Sequence[float], epsilon: float
+    lcu_coefficients: Sequence[float], **precision: float
```

Comment:
Eh, I think just epsilon: Optional[float] = None and sub_bit_prec: Optional[float] = None, is better here and just assert they're both not None.

Reply:
I've done this.